apps/builder/src/features/blocks/inputs/buttons/components/ButtonsBlockSettings.tsx

```diff
@@ -54,6 +57,11 @@ export const ButtonsBlockSettings = ({ options, onOptionsChange }: Props) => {
         }
         onCheckChange={updateIsSearchable}
       >
+        <SwitchWithLabel
+          label={t("blocks.inputs.settings.showAllOptionsByDefault.label")}
```

Comment:
```suggestion
          label="Default display buttons?"
```

Remove all the translations, this is done separately with Tolgee

Reply:
Done ✅